app/src/androidTest/java/net/skyscanner/backpack/compose/cardbutton/BpkCardButtonTest.kt

```diff
@@ -0,0 +1,98 @@
+package net.skyscanner.backpack.compose.cardbutton
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.IntSize
+import net.skyscanner.backpack.BpkSnapshotTest
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import org.junit.Assume.assumeTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+
+@RunWith(Parameterized::class)
+class BpkCardButtonTest(flavor: Flavor) : BpkSnapshotTest() {
+
+  private val type: BpkCardButtonType = flavor.type
+  private val size: BpkCardButtonSize = flavor.size
+  private val style: BpkCardButtonStyle = flavor.style
+
+  @Test
+  fun default() {
+    assumeTrue(style == BpkCardButtonStyle.Default)
```

Comment:
looks like all three tests always assume their style. does the style need to be part of the paramiterization?

Reply:
fixed it